examples/hello-world/hello-tf/src/hello-tf_fl.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tf_net import TFNet
+
+import nvflare.client as flare
+
+WEIGHTS_PATH = "./tf_model.weights.h5"
+
+
+def main():
+    flare.init()
+
+    sys_info = flare.system_info()
+    print(f"system info is: {sys_info}", flush=True)
+
+    model = TFNet()
+    model.build(input_shape=(None, 28, 28))
+    model.compile(
+        optimizer="adam", loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True), metrics=["accuracy"]
+    )
+    model.summary()
+
+    (train_images, train_labels), (
+        test_images,
+        test_labels,
+    ) = tf.keras.datasets.mnist.load_data()
+    train_images, test_images = (
+        train_images / 255.0,
+        test_images / 255.0,
+    )
+
+    # simulate separate datasets for each client by dividing MNIST dataset in half
+    client_name = sys_info["site_name"]
```

Comment:
I removed the prepare_data.sh but I'm not sure what to clarify or what do you mean by the dataset should be disjoint already

Reply:
I mean in reality different site has different dataset, otherwise defeat the purpose of FL.

So this "if site-1 read first half and if site-2 read second half" scenario does not fit into reality.

This is OK in hello-examples, just wondering if we need to clarify this.

Reply:
I see, there is just a single line just mentioning that it is split in half in the doc right now, I could add a little more